automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/sam/SamApiSpec.scala

```diff
@@ -24,9 +24,13 @@ class SamApiSpec extends FreeSpec with Matchers {
     Await.result(remove, 5.seconds)
   }
 
+  def findSaInGoogle(name: WorkbenchUserServiceAccountName): Option[WorkbenchUserServiceAccount] = {
+    val find = googleIamDAO.findServiceAccount(GoogleProject(Config.Projects.default), name)
+    Await.result(find, 1.minute)
```

Comment:
Is this wait time arbitrary? Do the tests consistently pass with this wait time?

Reply:
it is arbitrary - and way too long.  Let me reduce it.